### PR TITLE
Added namespace and type list

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**

### CODE REVIEW

File: src/dotnet/APIView/APIView/CodeFile.cs [hunk 1]
```diff
@@ -12,11 +15,29 @@ public class CodeFile
 
         public int Version { get; set; }
 
-        public CodeFileToken[] Tokens { get; set; } = Array.Empty<CodeFileToken>();
+        public CodeFileToken[] Tokens { get; set; } = Array.Empty<CodeFileToken>();
+
+        //[System.Text.Json.Serialization.JsonIgnore]
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/CodeFile.cs]: rem

File: src/dotnet/APIView/APIView/APIView.csproj [hunk 2]
```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis" Version="3.1.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.1.0" />
+    <PackageReference Include="System.Text.Json" Version="4.6.0-preview8.19405.3" />
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/APIView.csproj]: rem

File: src/dotnet/APIView/APIView/CodeFile.cs [hunk 3]
```diff
@@ -12,11 +15,29 @@ public class CodeFile
...
+        public List<NavigationItem> Navigation { get; set; } = new List<NavigationItem>();
 
         public override string ToString()
         {
             return new CodeFileRenderer().Render(this).ToString();
         }
     }
+
+    public class NavigationItem
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/CodeFile.cs]: class per file

File: src/dotnet/APIView/APIView/CodeFileBuilder.cs [hunk 4]
```diff
@@ -76,9 +76,21 @@ private IEnumerable<INamespaceSymbol> EnumerateNamespaces(IAssemblySymbol assemb
...
             var builder = new CodeFileTokensBuilder();
             foreach (var namespaceSymbol in EnumerateNamespaces(assemblySymbol))
             {
+                var nspace = new NavigationItem();
+                nspace.Text = namespaceSymbol.ContainingNamespace + "." + namespaceSymbol.Name;
+                nav.Add(nspace);
+
+                foreach(INamedTypeSymbol type in namespaceSymbol.GetTypeMembers())
+                {
+                    if (type.DeclaredAccessibility == Accessibility.Public | type.DeclaredAccessibility == Accessibility.Protected)
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/CodeFileBuilder.cs]: `IsAccessible(type)`

File: src/dotnet/APIView/APIViewWeb/Models/AssemblyModel.cs [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ﻿using ApiView;
 using System;
+using System.Collections.ObjectModel;
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Models/AssemblyModel.cs]: Undo

File: src/dotnet/APIView/APIView/CodeFile.cs [hunk 6]
```diff
@@ -12,11 +15,29 @@ public class CodeFile
 
         public int Version { get; set; }
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/CodeFile.cs]: Increment CurrentVersion

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml [hunk 7]
```diff
@@ -14,30 +14,49 @@
...
-</table>
-
+<table>
+    <tr>
+        <td class="namespace-view-panel">
+            <div class="namespace-view">
+                <ul class="nav_list">
+                @foreach (var ns in Model.AssemblyModel.AssemblyNode.Navigation) {
+                    <li>
+                        <div><img src="~/icons/down.png"> <b>@ns.Text</b> </div>
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml]: Right.png is not used, remove.

File: src/dotnet/APIView/APIViewWeb/wwwroot/css/site.css [hunk 8]
```diff
@@ -318,4 +318,34 @@ code-inner {
...
+    list-style-type:none;
+    padding-left: 0px;
+}
+
+.nav_list_children {
+    list-style-type: none;
+    padding-left: 23px;
+}
+
+.nav_expanded {
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/wwwroot/css/site.css]: not used.

File: src/dotnet/APIView/APIView/CodeFile.cs (hunk 1 above)
Comment (KrzysztofCwalina) [reply src/dotnet/APIView/APIView/CodeFile.cs]: will do

File: src/dotnet/APIView/APIView/APIView.csproj (hunk 2 above)
Comment (KrzysztofCwalina) [reply src/dotnet/APIView/APIView/APIView.csproj]: will do

File: src/dotnet/APIView/APIView/CodeFile.cs (hunk 3 above)
Comment (KrzysztofCwalina) [reply src/dotnet/APIView/APIView/CodeFile.cs]: will do

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml (hunk 7 above)
Comment (KrzysztofCwalina) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml]: I will be using it tomorrow.

**Pull request merged**